### PR TITLE
update siac version to show daemon version and siac version

### LINKED ISSUE #1490: siac version should show the daemon version as well
If siac is unable to connect to the daemon, it does not report an error and it does not show the daemon version.

### PR DESCRIPTION
**Pull request by johnathanhowell**
Fixes #1490

### CODE REVIEW

File: siac/daemoncmd.go [hunk 1]
```diff
@@ -27,13 +28,36 @@ var (
...
+
+// version prints the version of siac and siad.
+func versioncmd() {
+	var versioninfo daemonVersion
+	err := getAPI("/daemon/version", &versioninfo)
+	if err != nil {
+		fmt.Println("Could not get daemon version:", err)
+		return
+	}
+	fmt.Println("Sia Client v" + build.Version)
```
Comment (VoidingWarranties) [inline siac/daemoncmd.go]: If the Sia Daemon is not running, this command will exit before displaying `siac`'s version. Perhaps this line could be moved to the top of the func so as to always print the `siac` version.

Comment (johnathanhowell) [inline siac/daemoncmd.go]: yeah, that would be better.

File: siac/daemoncmd.go [hunk 2]
```diff
@@ -27,13 +28,36 @@ var (
...
 )
 
 type updateInfo struct {
 	Available bool   `json:"available"`
 	Version   string `json:"version"`
 }
 
+type daemonVersion struct {
+	Version string
+}
```
Comment (VoidingWarranties) [inline siac/daemoncmd.go]: Also, I don't think it's strictly necessary, but it might be good practice to have the JSON tags for the version field as well.

Comment (VoidingWarranties) [inline siac/daemoncmd.go]: Ah, right!

**Pull request merged**